### PR TITLE
Sample for release 1.10.0

### PR DESCRIPTION
**Pull request by wangchao1230**
# Description

Target changes: 

- Flex flow: 
  - basic chat notebook sample update: init with ModelConfiguration, aggregate
  - chat stream sample
- Prompty:
  - Prmpty format sample: text, json, all, stream

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/prompty/basic/prompty-quickstart.ipynb [hunk 1]
```diff
@@ -84,7 +84,9 @@
     "# load prompty as a flow\n",
     "f = Flow.load(\"basic.prompty\")\n",
     "# execute the flow as function\n",
```
Comment (wangchao1230) [inline examples/prompty/basic/prompty-quickstart.ipynb:86]: Change to Prompty.load() and add example on how we override parameters

File: examples/flex-flows/chat-stream/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Chat stream
+A chat flow defined using class entry that return output in stream mode. It demonstrates how to create a chatbot that can remember previous interactions and use the conversation history to generate next message.
+
+## Prerequisites
+
+Install promptflow sdk and other dependencies in this folder:
+```bash
+pip install -r requirements.txt
```
Comment (wangchao1230) [inline examples/flex-flows/chat-stream/README.md:8]: maybe add a sample using prompty in flex flow

Comment (D-W-) [reply examples/flex-flows/chat-stream/README.md:8]: Changed code evaluation sample to prompty + flex flow.

File: examples/flex-flows/basic/requirements.txt [hunk 3]
```diff
@@ -1,2 +1,8 @@
-promptflow-core
+promptflow-tracing @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-tracing
+promptflow-core @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-core
```
Comment (wangchao1230) [inline examples/flex-flows/basic/requirements.txt:2]: MARK

File: examples/flex-flows/chat-basic/requirements.txt [hunk 4]
```diff
@@ -1,2 +1,6 @@
-promptflow-core
-promptflow-tools
\ No newline at end of file
+promptflow-tracing @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-tracing
```
Comment (wangchao1230) [inline examples/flex-flows/chat-basic/requirements.txt:1]: MARK

File: examples/flex-flows/eval-checklist/requirements.txt [hunk 5]
```diff
@@ -1,2 +1,6 @@
-promptflow-core
-promptflow-tools
\ No newline at end of file
+promptflow-tracing @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-tracing
+promptflow-core @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-core
```
Comment (wangchao1230) [inline examples/flex-flows/eval-checklist/requirements.txt:2]: MARK

File: examples/flex-flows/eval-code-quality/requirements.txt [hunk 6]
```diff
@@ -1,2 +1,6 @@
-promptflow-core
-promptflow-tools
\ No newline at end of file
+promptflow-tracing @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-tracing
+promptflow-core @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-core
+promptflow-devkit @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-devkit
+promptflow-azure @ git+https://github.com/microsoft/promptflow/@clwan/sample_1_10#subdirectory=src/promptflow-azure
```
Comment (wangchao1230) [inline examples/flex-flows/eval-code-quality/requirements.txt:4]: MARK

File: examples/requirements.txt [hunk 7]
```diff
@@ -1,4 +1,5 @@
-promptflow[azure]
+--extra-index-url https://azuremlsdktestpypi.azureedge.net/promptflow/
+promptflow[azure]==1.10.0
```
Comment (wangchao1230) [inline examples/requirements.txt:2]: MARK

**Review by Stephen1993 — DISMISSED** [review]

**Review by 0mza987 — APPROVED** [review]

**Review by elliotzh — APPROVED** [review]

**Review by 0mza987 — DISMISSED** [review]

File: examples/prompty/chat-basic/chat-with-prompty.ipynb [hunk 8]
```diff
@@ -260,17 +305,17 @@
...
-    "    data=\"./data.jsonl\",  # path to the data file\n",
-    "    run=base_run,  # specify base_run as the run you want to evaluate\n",
-    "    column_mapping={\n",
-    "        \"messages\": \"${data.chat_history}\",\n",
-    "        \"question\": \"${data.question}\",\n",
-    "        \"answer\": \"${run.outputs.answer}\",\n",
-    "    },\n",
-    "    stream=True,\n",
-    ")"
+    "# eval_run = pf.run(\n",
```
Comment (elliotzh) [inline examples/prompty/chat-basic/chat-with-prompty.ipynb:308]: we should remove the section if it doesn't work? @0mza987

Comment (elliotzh) [reply examples/prompty/chat-basic/chat-with-prompty.ipynb:308]: and it works on my machine?
image

Comment (0mza987) [reply examples/prompty/chat-basic/chat-with-prompty.ipynb:308]: offline synced.

**Review by elliotzh — APPROVED** [review]

**Review by 0mza987 — DISMISSED** [review]

**Review by elliotzh — DISMISSED** [review]

**Review by 0mza987 — APPROVED** [review]

**Review by 0mza987 — APPROVED** [review]

**Review by elliotzh — APPROVED** [review]

**Pull request merged**